src/types.rs

```diff
@@ -46,6 +49,12 @@ impl EdgeKind {
     }
 }
 
+/// Python representation for [`EdgeKind`], the kinds of edges in a HUGR.
+#[cfg_attr(feature = "pyo3", pyclass)]
+#[repr(transparent)]
+#[derive(Clone, PartialEq, Eq, Debug, From, Into)]
+pub struct PyEdgeKind(EdgeKind);
```

Comment:
Why can't we just use `EdgeKind` for this?

Reply:
Because pyo3 doesn't like enums with non-unit variants. Wrapping it in a struct we hide the variants so it works alright (we'll need some python-level methods to access the internals later).